tests/internal/e2e/test/specs/vcast_build_env_failure.test.ts

```diff
@@ -0,0 +1,246 @@
+    await updateTestID();
+
+    const workbench = await browser.getWorkbench();
+    const activityBar = workbench.getActivityBar();
+    const explorerView = await activityBar.getViewControl("Explorer");
+    await explorerView?.openView();
+
+    const workspaceFolderSection =
+      await expandWorkspaceFolderSectionInExplorer("vcastTutorial");
+
+    const configFile = await workspaceFolderSection.findItem("CCAST_.CFG");
+    await configFile.openContextMenu();
+    await (await $("aria/Set as VectorCAST Configuration File")).click();
+    await (await workbench.openNotificationsCenter()).clearAllNotifications();
+  });
+
+  it("should create VectorCAST environment and succeed", async () => {
+    await updateTestID();
+
+    const workbench = await browser.getWorkbench();
+    const activityBar = workbench.getActivityBar();
+    const explorerView = await activityBar.getViewControl("Explorer");
+    const explorerSideBarView = await explorerView?.openView();
+
+    // Don't open folder again since its already expanded
+    const workspaceName = "vcastTutorial";
+    const workspaceFolderSection = await explorerSideBarView
+      .getContent()
+      .getSection(workspaceName.toUpperCase());
+
+    const managerCpp = await workspaceFolderSection.findItem("manager.cpp");
+
+    await managerCpp.openContextMenu();
+    await (await $("aria/Create VectorCAST Environment")).click();
+
+    // Making sure notifications are shown
+    await (await $("aria/Notifications")).click();
+
+    console.log("Notifications are shown");
+    // This will timeout if VectorCAST notification does not appear, resulting in a failed test
+    const vcastNotificationSourceElement = await $(
+      "aria/VectorCAST Test Explorer (Extension)"
+    );
+    const vcastNotification = await vcastNotificationSourceElement.$("..");
+
+    await (await vcastNotification.$("aria/Yes")).click();
+
+    console.log("Notifications clicked yes");
+    console.log(
+      "Waiting for clicast and waiting for environment to get processed"
+    );
+    await browser.waitUntil(
+      async () =>
+        (await (await bottomBar.openOutputView()).getText())
+          .toString()
+          .includes("Environment built Successfully"),
+      { timeout: TIMEOUT }
+    );
+
+    console.log("Finished creating vcast environment");
+  });
+
+  it("should delete .vscode folder", async () => {
+    await updateTestID();
+
+    const workbench = await browser.getWorkbench();
+    const activityBar = workbench.getActivityBar();
+    const explorerView = await activityBar.getViewControl("Explorer");
+    const explorerSideBarView = await explorerView?.openView();
+
+    const workspaceName = "vcastTutorial";
+    const workspaceFolderSection = await explorerSideBarView
+      .getContent()
+      .getSection(workspaceName.toUpperCase());
+
+    // Open.vscode folder
+    let vscodeFolderItem = await workspaceFolderSection.findItem(".vscode");
+    vscodeFolderItem.select();
```

Comment:
shouldn't we await here (the .select())?

Reply:
Added await in 26119a9b2016bce045ee2019c3eac106dafcea3b.